types/bull/index.d.ts

```diff
@@ -627,6 +642,12 @@ declare namespace Bull {
      */
     getJobs(types: string[], start?: number, end?: number, asc?: boolean): Promise<Array<Job<T>>>;
 
+    /**
+     * Returns a object with the logs according to the stard and end arguments. The returned count
```

Comment:
Think you meant start here.

Reply:
I understand that you'd better add newly defined functions down here, right?

Reply:
Oh, sorry. Think I was unclear... It was just a typo. `stard` should be `start`

Reply:
Oh, I understand. Thanks for comment!